altianalysis/run_difference_with_gpao.py

```diff
@@ -25,36 +26,55 @@ def get_tile_names(folder: Path) -> List[str]:
     return filenames
 
 
-def create_one_job_one_difference(store: Store, dir_in: Path, input_file: str, output: Path):
+def create_one_job_one_difference(store: Store, dir_in: Path, second_dir: Path | None, input_file: str, output: Path):
     job_name = f"difference_{input_file}"
-    command = f"""
-docker run -t --rm --userns=host
--v {store.to_unix(dir_in)}:/input
--v {store.to_unix(output)}:/output
-ghcr.io/ignf/altianalysis:{__version__}
-python -m altianalysis.compute_difference
---dtm_lidar_file /input/{input_file}
---name_save_out /output/{input_file}
-"""
+    if second_dir:
+        secondary_file = second_dir / input_file
+
+        if not os.path.exists(secondary_file):
+            raise FileNotFoundError(f"Secondary elevation does not exist to compute difference for {input_file}")
+
+        command = f"""
+    docker run -t --rm --userns=host
+    -v {store.to_unix(dir_in)}:/input
+    -v {store.to_unix(second_dir)}:/second_input
+    -v {store.to_unix(output)}:/output
+    ghcr.io/ignf/altianalysis:{__version__}
+    python -m altianalysis.compute_difference
+    --primary_elevation_file /input/{input_file}
+    --second_elevation_file /second_input/{input_file}
+    --name_save_out /output/{input_file}
+    """
+    else:
+        command = f"""
+    docker run -t --rm --userns=host
+    -v {store.to_unix(dir_in)}:/input
+    -v {store.to_unix(output)}:/output
+    ghcr.io/ignf/altianalysis:{__version__}
+    python -m altianalysis.compute_difference
+    --primary_elevation_file /input/{input_file}
+    --name_save_out /output/{input_file}
+    """
```

Comment:
à refactoriser pour éviter les oublis de mise à jour ultérieurs : 
```suggestion
    if second_dir:
        secondary_file = second_dir / input_file

        if not os.path.exists(secondary_file):
            raise FileNotFoundError(f"Secondary elevation does not exist to compute difference for {input_file}")

         second_input_mount  = f" -v {store.to_unix(second_dir)}:/second_input"
         second_input_param = f"--second_elevation_file /second_input/{input_file}"

    else:
        second_input_mount = ""
        second_input_param = ""
      
      
        command = f"""
    docker run -t --rm --userns=host
    -v {store.to_unix(dir_in)}:/input
   {second_input_mount}
    -v {store.to_unix(output)}:/output
    ghcr.io/ignf/altianalysis:{__version__}
    python -m altianalysis.compute_difference
    --primary_elevation_file /input/{input_file}
    {second_input_param}
    --name_save_out /output/{input_file}

```